viewer/src/glsl/base/updateFragmentColor.glsl

```diff
@@ -2,16 +2,32 @@
 #pragma glslify: hsv2rgb = require('../color/hsv2rgb.glsl')
 #pragma glslify: packIntToColor = require('../color/packIntToColor.glsl')
 
+uniform sampler2D colorDataTexture;
+
 const int RenderTypeColor = 1;
 const int RenderTypeNormal = 2;
 const int RenderTypeTreeIndex = 3;
 const int RenderTypePackColorAndNormal = 4;
+const float dataTextureWidth = 2048.0;
+const float dataTextureHeight = 2048.0;
 
 vec3 packNormalToRGB( const in vec3 normal ) {
     return normalize( normal ) * 0.5 + 0.5;
 }
 
+// TODO remove input color
```

Comment:
Why?

Reply:
Ooops, it was a stray comment left behind.